src/combinatorics/quiver/basic.lean

```diff
@@ -61,7 +61,7 @@ lemma ext {V : Type u} [quiver.{v₁} V] {W : Type u₂} [quiver.{v₂} W]
   {F G : prefunctor V W}
   (h_obj : ∀ X, F.obj X = G.obj X)
   (h_map : ∀ (X Y : V) (f : X ⟶ Y),
-           F.map f = eq.rec_on (h_obj Y).symm (eq.rec_on (h_obj X).symm (G.map f))) : F = G :=
+           F.map f = eq.rec_on (h_obj X).symm (eq.rec_on (h_obj Y).symm (G.map f))) : F = G :=
```

Comment:
Why swap `X` and `Y` here?

Reply:
Can't say for sure, it's been a long time.

Reply:
Could you investigate and either revert or add a comment?

Reply:
I'm kind of out of the loop on all this, but after trying it out in gitpod, it seems swapping `X` and `Y` works for the `ext` lemma itself, but then `of_bijective` and further lemmas in the `iso` file get failing proof, which I couldn't debug.
A comment as to that is what I should do?

Reply:
( ping @semorrison )